[2025-03-10T00:11:27.272+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_maestro.load_data manual__2025-03-10T00:02:32.496605+00:00 [queued]>
[2025-03-10T00:11:27.297+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_maestro.load_data manual__2025-03-10T00:02:32.496605+00:00 [queued]>
[2025-03-10T00:11:27.297+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-03-10T00:11:27.334+0000] {taskinstance.py:1350} INFO - Executing <Task(TriggerDagRunOperator): load_data> on 2025-03-10 00:02:32.496605+00:00
[2025-03-10T00:11:27.349+0000] {standard_task_runner.py:57} INFO - Started process 857 to run task
[2025-03-10T00:11:27.357+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_maestro', 'load_data', 'manual__2025-03-10T00:02:32.496605+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/orchestrator.py', '--cfg-path', '/tmp/tmpc73odihy']
[2025-03-10T00:11:27.360+0000] {standard_task_runner.py:85} INFO - Job 48: Subtask load_data
[2025-03-10T00:11:27.499+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_maestro.load_data manual__2025-03-10T00:02:32.496605+00:00 [running]> on host d7c1c9924032
[2025-03-10T00:11:27.734+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_maestro' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:02:32.496605+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-10T00:02:32.496605+00:00'
[2025-03-10T00:11:27.919+0000] {trigger_dagrun.py:201} INFO - Waiting for load_data_mysql on 2025-03-10T00:11:27.743782+00:00 to become allowed state ['success'] ...
[2025-03-10T00:12:28.020+0000] {trigger_dagrun.py:210} INFO - load_data_mysql finished with allowed state success
[2025-03-10T00:12:28.063+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=dag_maestro, task_id=load_data, execution_date=20250310T000232, start_date=20250310T001127, end_date=20250310T001228
[2025-03-10T00:12:28.157+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-03-10T00:12:28.293+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
